packages/framework/src/Component/Cron/CronFacade.php

```diff
@@ -26,19 +27,27 @@ class CronFacade
      */
     protected $cronModuleFacade;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Mail\Mailer
+     */
+    protected $mailer;
+
     /**
      * @param \Symfony\Bridge\Monolog\Logger $logger
      * @param \Shopsys\FrameworkBundle\Component\Cron\Config\CronConfig $cronConfig
      * @param \Shopsys\FrameworkBundle\Component\Cron\CronModuleFacade $cronModuleFacade
+     * @param \Shopsys\FrameworkBundle\Model\Mail\Mailer $mailer
      */
     public function __construct(
```

Comment:
Shouldn't there be upgrade notes for this change?

Reply:
`fix constructor of FrameworkBundle/Model/Mail/Mailer if you extended it somehow`. ???